src/quest_manager/templatetags/quest_tags.py

```diff
@@ -14,3 +16,33 @@ def is_hidden(quest, user):
     if not user or not quest:
         return None
     return user.profile.is_quest_hidden(quest)
+
+
+@register.simple_tag(takes_context=True)
+def can_export(context):
+    """
+    Template tag to determine if the current user is allowed to export a quest
+    to the library.
+
+    Returns True if:
+    - The current user is the deck owner for this site, or
+    - The site configuration allows staff to export quests and user is staff.
+
+    Returns False if:
+    - The current schema is the library schema (i.e., user is already in the library), or
+    - The request or user is not available or not authenticated.
+    """
+    request = context.get('request')
+    if not request or not getattr(request, 'user', None) or not request.user.is_authenticated:
```

Comment:
how is an unauthenticated user accessing this page? how would there not be a request?  What are all these things checking?  Are they all tested?